package.json

```diff
@@ -5,23 +5,27 @@
   "main": "lib/standalone-single-spa.js",
   "scripts": {
     "test": "jest",
+    "build:types": "tsc",
+    "prepublishOnly": "pnpm run build:types",
     "format": "prettier --write .",
     "lint": "eslint lib"
   },
+  "types": "types/standalone-single-spa.d.ts",
   "files": [
     "lib"
   ],
   "devDependencies": {
-    "@types/jest": "^26.0.15",
-    "eslint": "^7.14.0",
+    "@types/jest": "^26.0.20",
+    "eslint": "^7.17.0",
     "eslint-config-node-important-stuff": "^1.1.0",
-    "html-webpack-plugin": "^4.5.0",
-    "husky": "^4.3.0",
+    "html-webpack-plugin": "^5.0.0-beta.4",
```

Comment:
Necessary because of https://github.com/jantimon/html-webpack-plugin/issues/1526